testsuite/MDAnalysisTests/analysis/test_helix_analysis.py

```diff
@@ -430,6 +434,13 @@ def test_residue_gaps_no_split(self, psf_ca):
         assert 'has gaps in the residues' in warnmsg
         assert 'Splitting into' not in warnmsg
 
+    def test_len_groups_short(self, psf_ca):
+        sel = 'resnum 161-168'
+        with pytest.warns(UserWarning, match='Fewer than 9 atoms found'):
+            ha = hel.HELANAL(psf_ca, select=sel)
+            ha.run()
```

Comment:
Does the warning require run to be called to be emitted? (i.e. can we speed up CI by not calling run?)

Reply:
Oh, sorry. Good catch! You're right. Thanks. :)